vertical-pod-autoscaler/pkg/recommender/input/cluster_feeder.go

```diff
@@ -49,7 +49,13 @@ import (
 	resourceclient "k8s.io/metrics/pkg/client/clientset/versioned/typed/metrics/v1beta1"
 )
 
-const defaultResyncPeriod time.Duration = 10 * time.Minute
+const (
+	scaleCacheLoopPeriod time.Duration = time.Hour
+	scaleCacheCleanup    time.Duration = time.Hour
```

Comment:
Would be nice to decide on one of lifetime and cleanup (both are used to describe the cache entry ttl AFAIU)

Reply:
Done.